src/ensembl/src/content/app/species-selector/components/species-commit-button/SpeciesCommitButton.tsx

```diff
@@ -17,27 +17,35 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
+import useSpeciesSelectorAnalytics from 'ensemblRoot/src/content/app/species-selector/hooks/useSpeciesSelectorAnalytics';
+
 import {
   hasCurrentSpecies,
-  canCommitSpecies
+  canCommitSpecies,
+  getSelectedItem
 } from 'src/content/app/species-selector/state/speciesSelectorSelectors';
 import { commitSelectedSpeciesAndSave } from 'src/content/app/species-selector/state/speciesSelectorActions';
 
 import { PrimaryButton } from 'src/shared/components/button/Button';
 
-import styles from './SpeciesCommitButton.scss';
-
+import { CurrentItem } from 'src/content/app/species-selector/state/speciesSelectorState';
 import { RootState } from 'src/store';
 
+import styles from './SpeciesCommitButton.scss';
+
 type Props = {
   hasCurrentSpecies: boolean; // tells whether a species has been selected and can be committed
+  selectedItem: CurrentItem | null;
```

Comment:
Can the `selectedItem` property do the job of `hasCurrentSpecies` now? If yes, then there is no reason for keeping the `hasCurrentSpecies` property; and `selectedItem` can also be renamed to something more specific (currentSpecies? preselectedSpecies?).

Reply:
That's done. Changes it to `currentSpecies`.